https://issues.redhat.com/browse/ACM-14268 (#6974)
open-cluster-management/rhacm-docs: release_notes/whats_new.adoc

[#dr4hub-whats-new]
== Backup and restore

> * You can now use a restore hub as a cluster with resources that a user created before the restore operation. You can tag user created resources on the restore hub with the `velero.io/backup-name: backupName` label. See link:../business_continuity/backup_restore/backup_existing_clusters.adoc#backup-existing-clusters[Understanding outcomes for existing managed clusters during the restore process] and link:../backup_restore/backup_tag_resources.adoc#backup-tagging-resources[Tagging resources].

Reviewer: This entry is confusing. Should it be the following:

```suggestion
* You can now use an existing hub cluster as a restore hub cluster by tagging user-created resources on the restore hub cluster with the `velero.io/backup-name: backupName` label. See link:../business_continuity/backup_restore/backup_existing_clusters.adoc#backup-existing-clusters[Understanding outcomes for existing managed clusters during the restore process] and link:../backup_restore/backup_tag_resources.adoc#backup-tagging-resources[Tagging resources].
```
